### PR TITLE
[DOCFIX] Update Alluxio master doc

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

**Comment by peisun1115** [comment]
I can translate this to Chinese if the English version looks good.

File: docs/en/Architecture.md [hunk 1]
```diff
@@ -21,7 +21,13 @@ Alluxio's design uses a single master and multiple workers. At a very high level
...
+#### Secondary Master
+The secondary master replays journals written by the primary master and periodically writes checkpoints. It does not process any requests from any Alluxo components.
+#### Master Deployment
+Alluxio may be deployed in one of two master modes, [simple mode](Running-Alluxio-Locally.html) or [high availablity mode](Running-Alluxio-Fault-Tolerant.html). There must be one and only one primary master in both of the simple mode and the high availability mode. The simple mode can at most have one secondary master which cannot be promoted to the primary master. The high availability mode can have zero or more secondary masters, one of which can be promoted to the primary master if the primary master fails.
```
Comment (calvinjia) [inline docs/en/Architecture.md]: Please make each line at most 100 characters

File: docs/en/Architecture.md [hunk 2]
```diff
@@ -21,7 +21,13 @@ Alluxio's design uses a single master and multiple workers. At a very high level
 
 ### Master
 
-Alluxio may be deployed in one of two master modes, [single master](Running-Alluxio-Locally.html) or [fault tolerant mode](Running-Alluxio-Fault-Tolerant.html). The master is primarily responsible for managing the global metadata of the system, for example, the file system tree. Clients may interact with the master to read or modify this metadata. In addition, all workers periodically heartbeat to the master to maintain their participation in the cluster. The master does not initiate communication with other components; it only interacts with other components by responding to requests.
+Alluxio has two types of masters, primary master or secondary master.
```
Comment (calvinjia) [inline docs/en/Architecture.md]: `and can be deployed in one of two modes ...` ?

File: docs/en/Architecture.md (hunk 1 above)
Comment (calvinjia) [inline docs/en/Architecture.md]: `one of which can be promoted` -> `one of which will be promoted`

Comment (peisun1115) [inline docs/en/Architecture.md]: done.

Comment (peisun1115) [inline docs/en/Architecture.md]: done

File: docs/en/Architecture.md (hunk 2 above)
Comment (peisun1115) [inline docs/en/Architecture.md]: done

**Comment by calvinjia** [comment]
@peisun1115 Could you use 100 line character length, it looks like its around 70 at the moment.

**Comment by peisun1115** [comment]
@calvinjia PTAL. thanks

File: docs/cn/Architecture.md [hunk 3]
```diff
@@ -11,22 +11,43 @@ priority: 1
 
 ## Alluxio的应用场景
 
-由于Alluxio的设计以内存为中心，并且是数据访问的中心，所以Alluxio在大数据生态圈里占有独特地位，它居于大数据存储（如：Amazon S3，Apache HDFS和OpenStack Swift等）和大数据计算框架（如Spark，Hadoop Mapreduce）之间。对于用户应用和计算框架，无论其是否运行在相同的计算引擎之上，Alluxio都可以作为底层来支持数据的访问、快速存储，以及多任务的数据共享和本地化。因此，Alluxio可以为那些大数据应用提供一个数量级的加速，同时它还提供了通用的数据访问接口。对于底层存储系统，Alluxio连接了大数据应用和传统存储系统之间的间隔，并且重新定义了一组面向数据使用的工作负载程序。因Alluxio对应用屏蔽了底层存储系统的整合细节，所以任何底层存储系统都可以支撑运行在Alluxio之上的应用和框架。此外Alluxio可以挂载多种底层存储系统，所以它可以作为统一层为任意数量的不同数据源提供服务。
+由于Alluxio的设计以内存为中心，并且是数据访问的中心，所以Alluxio在大数据生态圈里占有独特地位，它居于大数据存储（如：Amazon S3，Apache HDFS和OpenStack
```
Comment (peisun1115) [inline docs/cn/Architecture.md]: Not sure how to limit the line length for Chinese. It looks like a Chinese character is counted as one character in IntelliJ but two in github.

File: docs/cn/Architecture.md [hunk 4]
```diff
@@ -11,22 +11,43 @@ priority: 1
...
 
 ## Alluxio的组件
 
-Alluxio的设计使用了单Master和多Worker的架构。从高层的概念理解，Alluxio可以被分为三个部分，[Master](#Master)，[Worker](#Worker)和[Client](#Client)。Master和Worker一起组成了Alluxio的服务端，它们是系统管理员维护和管理的组件。Client通常是应用程序，如Spark或MapReduce作业，或者Alluxio的命令行用户。Alluxio用户一般只与Alluxio的Client组件进行交互。
+Alluxio的设计使用了单Master和多Worker的架构。从高层的概念理解，Alluxio可以被分为三个部分，[Master](#Master)，[Worker](#Worker)和[Client](#Client)。
+Master和Worker一起组成了Alluxio的服务端，它们是系统管理员维护和管理的组件。Client通常是应用程序，如Spark或MapReduce作业，或者Alluxio的命令行用户。
+Alluxio用户一般只与Alluxio的Client组件进行交互。
 
 #### Master
+Alluxio Master有主从两中模式。
```
Comment (ifcharming) [inline docs/cn/Architecture.md]: s/两中/两种

File: docs/cn/Architecture.md [hunk 5]
```diff
@@ -11,22 +11,43 @@ priority: 1
...
+##### 主Master
+主Master主要负责处理全局的系统元数据，例如，文件系统树。Client可以通过与Master的交互来读取或修改元数据。此外所有的Worker会周期性地发送心跳给主Master，
+来确保它们还参与在Alluxio集群中。主Master不会主动发起与其他组件的通信，它只是以回复请求的方式与其他组件进行通信。一个Alluxio集群只有一个主Master。
+
+##### 从Master
+从Master不断的读取并处理主Master写的日志。同时从Master会周期性的把所有的状态写入日志。从Master不处理任何请求。
+
+##### Master部署
+Alluxio Master有[简单](#Running-Alluxio-Locally.html)和[高可用性](#Running-Alluxio-Fault-Tolerant.html)
+两种部署模式。这两种模式都只有一个主Maste。简单模式最多只会有一个从Master，而且这个从Master不会被转换为主Maste。高可用性模式可以有零个或者多个从Maste。
```
Comment (ifcharming) [inline docs/cn/Architecture.md]: s/Maste/Master

File: docs/cn/Architecture.md (hunk 4 above)
Comment (peisun1115) [inline docs/cn/Architecture.md]: done

File: docs/cn/Architecture.md (hunk 5 above)
Comment (peisun1115) [inline docs/cn/Architecture.md]: done

File: docs/cn/Architecture.md (hunk 3 above)
Comment (calvinjia) [inline docs/cn/Architecture.md]: 1 to 1 is fine.

**Pull request merged**